sources/packages/backend/libs/test-utils/src/factories/disbursement-schedule.ts

```diff
@@ -34,3 +41,24 @@ export function createFakeDisbursementSchedule(relations?: {
   schedule.disbursementScheduleStatus = DisbursementScheduleStatus.Pending;
   return schedule;
 }
+
+/**
+ * Creates disbursement schedule awards based on the given schedule and disbursement number.
+ * @param disbursementSchedule Disbursement schedule containing disbursement values.
+ * @param disbursementNumber Number associated with the disbursement schedule (e.g., 1 for the first, 2 for the second).
+ * @returns Disbursement awards with keys like 'disbursement1code' and values as amounts.
+ */
+export function createDisbursementScheduleAwards(
```

Comment:
Is this generating the API payload for the disbursement awards, right?
This is an API-specific method for a specific controller. Maybe it can be restricted to that controller's methods or even a local method if it fits the current scenario.